app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com_list = command.split()
```

Comment:
Consider using more descriptive variable names. Instead of `com_list`, something like `command_parts` would be clearer and more descriptive.